toqito/matrix_props/tests/test_is_anti_hermitian.py

```diff
@@ -0,0 +1,23 @@
+"""Test is_hermitian."""
+
+import numpy as np
+
+from toqito.matrix_props import is_anti_hermitian
+
+
+def test_is_hermitian():
+    """Test if matrix is Hermitian."""
+    mat = np.array([[2j, -1 + 2j, 4j], [1 + 2j, 3j, -1], [4j, 1, 1j]])
+    np.testing.assert_equal(is_anti_hermitian(mat), True)
+
+
+def test_is_non_hermitian():
+    """Test non-Hermitian matrix."""
+    mat = np.array([[1, 2, 3], [4, 5, 6], [7, 8, 9]])
+    np.testing.assert_equal(is_anti_hermitian(mat), False)
+
+
+def test_is_hermitian_not_square():
+    """Input must be a square matrix."""
+    mat = np.array([[-1, 1, 1], [1, 2, 3]])
+    np.testing.assert_equal(is_anti_hermitian(mat), False)
```

Comment:
Minor nit: I think all 3 tests could be condensed into one with `@pytest.mark.parametrize("test_input,expected_bool"` where `test_input = mat` and `expected_bool = expected result`.

https://docs.pytest.org/en/7.1.x/how-to/parametrize.html

Reply:
I saw that you approved the PR, do you still want me to do this modification? I don't want to waste your time by requesting review after review!

Reply:
Yes, I approved the PR. But go ahead and resolve this suggestion. It's OK, I can review the newer changes afterward.